src/System.cc

```diff
@@ -1410,6 +1410,10 @@ void System::SaveAtlasAsOsaWithTimestamp(string pathSaveFileName) {
     mpAtlas->SaveAtlas(pathSaveFileName, strVocabularyName, strVocabularyChecksum);
 }
 
+void System::RenameShutdownSaveAtlas(string pathSaveFileName){
```

Comment:
made this function to change the last map saved into the current timestamp, which should prevent any data from being rerun if testing using loaded maps

Reply:
not sure if I can just put this in the header file since its only 1 line? Not sure if thats bad practice or somethin

Reply:
I'm not worried about having this in the header, but I'm a little worried of changing what's in System.SaveAtlasToFile. Why have this field at all if we're going to change the filename?

Reply:
I am fine with removing the field from the yaml file generated in rdk if we plan on using this, in the [slam repo PR](https://github.com/viamrobotics/slam/pull/62#discussion_r992469014) I show how I would want to use the new function